wp-content/themes/core/blocks/core/posttemplate/style.pcss

```diff
@@ -0,0 +1,50 @@
+/**
+ * Styles specific to this block
+ */
+
+/**
+ * Vertical (list layout) post template spacing
+ */
+.wp-block-post-template:not(.is-layout-grid) .wp-block-post + .wp-block-post {
+	margin-top: var(--spacer-40);
```

Comment:
Can we use `gap` preferably or `margin-block-start`?

Reply:
`gap` isn't available in this context. In 6.3 when the post template block isn't set to be a grid, it doesn't get any sort of flex layout at all. Just plain block layout. I've updated to `margin-block-start`, though.